escrow/etcd_storage_test.go

```diff
@@ -0,0 +1,215 @@
+package escrow
+
+import (
+	"bytes"
+	"math/big"
+	"net"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/singnet/snet-daemon/config"
+	"github.com/singnet/snet-daemon/etcddb"
+	"github.com/singnet/snet-daemon/handler"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+var etcdPaymentHandler escrowPaymentHandler
+var etcdStorageMock etcdStorageMockType
+
+type etcdStorageMockType struct {
```

Comment:
I would propose renaming it to ```cleanableEtcdStorage``` because ```etcdStorageMock``` means some fake etcd storage, but test itself tests real storage as far as I see. 

Solution which seems for me even better is to add some methods to cleanup storage using etcd API. 

I agree to postpone fixes till next PR.

Reply:
I renamed the etcdStorageMock in the branch:
https://github.com/stellarspot/snet-daemon/commit/7d79313a742e6a9d0899f3b457b4bd90ad800cd3